src/core/main.c

```diff
@@ -1600,9 +1600,9 @@ static void initialize_clock(void) {
 
         r = clock_apply_epoch();
         if (r < 0)
-                log_error_errno(r, "Current system time is before build time, but cannot correct: %m");
+                log_error_errno(r, "Current system time diverges from epoch time, but cannot correct: %m");
```

Comment:
Why this change? The previous version is more accurate

Reply:
Because the system time can be not only **before**, but also far after than the build time(on some systems the rtc reset to the future), and I need to detect this and adjust the system time.

Reply:
i don't like this error message. I should clearly say what was tried here, i.e. jump back or forward, given that the error situation is quite a different once.

i.e. change clock_apply_epoch() to return backward or forward jump info somehow. Or call it twice, once for backward and once for forward jumps, with a new param that discerns the two oprations.